server/tests/src/test/java/org/infinispan/server/functional/RestLoggingResource.java

```diff
@@ -0,0 +1,85 @@
+package org.infinispan.server.functional;
+
+import static org.infinispan.server.security.Common.sync;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.server.test.junit4.InfinispanServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerTestMethodRule;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * @since 11.0
+ */
+public class RestLoggingResource {
+
+   @ClassRule
+   public static InfinispanServerRule SERVERS = ClusteredIT.SERVERS;
+
+   @Rule
+   public InfinispanServerTestMethodRule SERVER_TEST = new InfinispanServerTestMethodRule(SERVERS);
+
+   private final ObjectMapper mapper = new ObjectMapper();
+
+   @Test
+   public void testListLoggers() throws Exception {
+      RestClient client = SERVER_TEST.rest().create();
+      RestResponse response = sync(client.raw().get("/rest/v2/logging/loggers"));
+      JsonNode loggers = mapper.readTree(response.getBody());
+      assertTrue(loggers.size() > 0);
+   }
+
+   @Test
+   public void testListAppenders() throws Exception {
+      RestClient client = SERVER_TEST.rest().create();
+      RestResponse response = sync(client.raw().get("/rest/v2/logging/appenders"));
+      JsonNode appenders = mapper.readTree(response.getBody());
+      assertEquals(2, appenders.size());
+   }
+
+   @Test
+   public void testManipulateLogger() throws Exception {
```

Comment:
can you duplicate the test using `RestLoggingClient`? It would avoid issues in `RestLoggingClient` :)

Reply:
Done